### PR TITLE
Added Feature for Bulk Generating Certificates

### PR DESCRIPTION
**Pull request by saivittalb**
Perfectly added feature which makes organizations/individual issuers to bulk generate certificates by uploading CSV or Excel files up to 5MB 🎉

To test this feature, run the project locally using ```ng serve``` and then change the address to ```localhost:4200/organization```

Workflow of the feature 👇

1. Uploaded file is sent to the ```baseUrl``` which is ```https://jbossnewserver.herokuapp.com/``` for validation.
2. The validated file is returned back and is displayed in the ```ag-grid``` or ```Folders```, else error is displayed stating```invalid file```.

Preview of the default page until server works 👇

screen shot 2018-12-12 at 5 52 17 pm

### CODE REVIEW

File: src/environments/environment.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  baseUrl: 'https://jbossnewserver.herokuapp.com'
 };
```
Comment (nikitadhiman) [inline src/environments/environment.ts]: Did you test this base URL? '/' is missing

Comment (saivittalb) [reply src/environments/environment.ts]: No... you didn’t test this URL! I found it in the issues section where it was stated as a new server and I used it!

Comment (saivittalb) [reply src/environments/environment.ts]: Done adding '/' ✅

File: src/app/organization/organization.component.html [hunk 2]
```diff
@@ -14,32 +14,28 @@ <h4> Download </h4>
...
+      class="ag-theme-balham"
+      *ngIf="!error"
+      [rowData]="rowData"
+      [enableSorting]="true"
+      [enableFilter]="true"
+      [columnDefs]="columnDefs"
+      [enableColResize] = "true"
+      (gridReady)="onGridReady($event)"
+      >
+    </ag-grid-angular>
```
Comment (nikitadhiman) [inline src/app/organization/organization.component.html]: Just one thing, we are thinking of merging it. Please retain cards as well as list for this component.

Comment (saivittalb) [reply src/app/organization/organization.component.html]: I will work on it and submit it by tomorrow's afternoon!

Comment (nikitadhiman) [reply src/app/organization/organization.component.html]: Sure, take your time

**Comment by saivittalb** [comment]
Done ✅

Now, even the folders list the certificate data from the backend! 👇

screen shot 2018-12-12 at 5 57 44 pm

But the default response until the JBoss server works is this 👇

screen shot 2018-12-12 at 5 52 17 pm

P.S - I have also added ```navbar``` to the login page and ```routerLink``` to the SignUp button in the ```navbar```. Some changes were made! 🎉

File: src/app/organization/organization.component.html (hunk 2 above)
Comment (saivittalb) [reply src/app/organization/organization.component.html]: Done making changes requested ✅
Sorry, I was super late... I had a surprise work given by the school which had to be completed!
Please check the changes I made with my new commit below👇

**Pull request closed without merging**